paddle/fluid/framework/ir/mkldnn/quant_transpose2_dequant_onednn_fuse_pass.h

```diff
@@ -28,8 +28,10 @@ class FuseQuantTranspose2DequantOneDNNPass : public FusePassBase {
 
  protected:
   void ApplyImpl(Graph *graph) const override;
-  void FuseQuantizeTranspose2(Graph *graph) const;
-  void FuseTranspose2Dequantize(Graph *graph) const;
+  void FuseQuantizeTranspose2(Graph *graph,
```

Comment:
If it fuses both transpose2 and fused_transpose shouldn't it be also called `FuseQuantizeTranspose` (without 2)? The same with Dequantize